README.md

```diff
@@ -20,6 +20,12 @@ pipenv install
 
 ## Configure the script
 
+Get an updated list of repos:
+
+```
+curl https://docs.publishing.service.gov.uk/repos.json | jq -r '.[] | {app_name} | join(" ")' | sed -e "s/(.*)/'alphagov/\1',/"
```

Comment:
This won't work as-is, because the `/` after `alphagov` needs to be escaped or it'll be treated as a delimiter:

```suggestion
curl https://docs.publishing.service.gov.uk/repos.json | jq -r '.[] | {app_name} | join(" ")' | sed -e "s/(.*)/'alphagov\/\1',/"
```

Alternatively, use some other character as delimiter:

```suggestion
curl https://docs.publishing.service.gov.uk/repos.json | jq -r '.[] | {app_name} | join(" ")' | sed -e "s!(.*)!'alphagov/\1',!"
```
But also, jq can do this without sed at all:

```suggestion
curl https://docs.publishing.service.gov.uk/repos.json | jq -r ".[].app_name | \"'alphagov/\" + . + \"',\""
```

(though it's a bit messy because of nested quoting.)

Reply:
Ahh that's not the right command, what I actually used was
curl https://docs.publishing.service.gov.uk/repos.json | jq -r '.[] | {app_name} | join(" ")' | sed -e "s/\(.*\)/'alphagov\/\1',/"